charts/fission-core/templates/NOTES.txt

```diff
@@ -0,0 +1,37 @@
+1. Install the client CLI.
+
+Mac:
+  $ curl http://fission.io/mac/fission > fission && chmod +x fission && sudo mv fission /usr/local/bin/
```

Comment:
We've switched to using github releases for the CLIs.

Can you update this to the command from INSTALL.md:
```
curl -Lo fission https://github.com/fission/fission/releases/download/nightly20170705/fission-cli-osx && chmod +x fission && sudo mv fission /usr/local/bin/
```